src/current/v23.2/live-migration-service.md

```diff
@@ -0,0 +1,612 @@
+      secret:
+        secretName: cockroach-key
+  sslVolumeMounts:
+    - mountPath: "/app/certs"
+      name: source-ca
+      readOnly: true
+    - mountPath: "/app/certs"
+      name: source-cert
+      readOnly: true
+    - mountPath: "/app/certs"
+      name: source-key
+      readOnly: true
+    - mountPath: "/app/certs"
+      name: cockroach-ca
+      readOnly: true
+    - mountPath: "/app/certs"
+      name: cockroach-cert
+      readOnly: true
+    - mountPath: "/app/certs"
+      name: cockroach-key
+      readOnly: true
+~~~
+
+Specify the corresponding client certificates and keys in the [database connection strings](#source-and-target-connections). For example, within `config.json` in the [external secret](#manage-external-secret):
+
+~~~ json
+config.json: |
+  {
+    "INIT_SOURCE": "mysql://{username}:{password}@{protocol}({host}:{port})/{database}?sslmode=verify-full?sslrootcert=path/to/mysql.ca&sslcert=path/to/mysql.crt&sslkey=path/to/mysql.key",
+    "INIT_TARGET": "postgresql://{username}:{password}@{host}:{port}/{database}?sslmode=verify-full?sslrootcert=path/to/ca.crt&sslcert=path/to/client.username.crt&sslkey=path/to/client.username.key"
+  }
+~~~
+
+#### Configure orchestrator and client certificates
+
+Mount the orchestrator certificate, key, and (optional) CA certificate to separate volumes. Cockroach Labs recommends mounting certificates to `/app/certs`.
+
+For example:
+
+~~~ yaml
+orchestrator:
+  sslVolumes:
+    - name: orch-ca
+      secret:
+        secretName: orch-ca
+    - name: orch-cert
+      secret:
+        secretName: orch-cert
+    - name: orch-key
+      secret:
+        secretName: orch-key
+  sslVolumeMounts:
+    - mountPath: "/app/certs"
+      name: orch-ca
+      readOnly: true
+    - mountPath: "/app/certs"
+      name: orch-cert
+      readOnly: true
+    - mountPath: "/app/certs"
+      name: orch-key
+      readOnly: true
+~~~
+
+Set the [corresponding config keys](#cert-var) to the same mount path. For example, within `config.json` in the [external secret](#manage-external-secret):
+
+~~~ yaml
+config.json: |
+  {
+    "CA_TLS_CERT": "/app/certs",
+    "TLS_CERT": "/app/certs",
+    "TLS_KEY": "/app/certs",
+  }
+~~~
+
+This also requires that you create and specify a CLI client certificate, key, and (optional) CA certificate. It's easiest to specify these as environment variables in the shell that is running `molt-lms-cli`:
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+export CLI_TLS_CA_CERT="{path-to-cli-ca-cert}"
+~~~
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+export CLI_TLS_CLIENT_CERT="{path-to-cli-client-cert}"
+~~~
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+export CLI_TLS_CLIENT_KEY="{path-to-cli-client-key}"
+~~~
+
+## `molt-lms-cli`
+
+The `molt-lms-cli` command-line interface is used to inspect the LMS instances and [perform cutover](#perform-a-cutover).
+
+To install `molt-lms-cli`, [download the binary](https://molt.cockroachdb.com/lms/cli/).
```

Comment:
This will have to be a direct link to the binaries. First we'll start with:
https://molt.cockroachdb.com/lms/cli/molt-lms-cli-0.1.0.darwin-arm64.tgz
https://molt.cockroachdb.com/lms/cli/molt-lms-cli-0.1.0.darwin-amd64.tgz